.editorconfig

```diff
@@ -2,8 +2,19 @@
 root = true
 
 [*]
+insert_final_newline = true
+charset = utf-8
+
+[*.sln]
+end_of_line = crlf
+indent_style = tab
+
+[*.csproj]
+end_of_line = crlf
 indent_style = space
 indent_size = 2
```

Comment:
And while we're at it, we could also just format everything with 4 spaces then instead of having some files be tabs and some files 2 spaces and some files 4 spaces.
I might've chosen the indent style, but I still stand behind having a single universal one and not 3 different ones.

Reply:
sure, 
what are yalls thoughts on swapping to a single tab?

Reply:
Single tab is my preferred indent style, so I hope you're not joking :'P

Reply:
My only beef with tabs is that some text renderers do something ridiculous and default to a tab width of 8. Github diffs USED to do this, at least. IDK if they still do, or if they can actually parse editorconfig or what now.

That being said I just want this debacle to stop causing huge diffs and blocking other PRs so I am happy to go with literally anything.

Reply:
if tabs still cause issues 4 spaces are fine.